xdem/spatial_tools.py

```diff
@@ -106,39 +106,24 @@ def merge_bounding_boxes(bounds: list[rio.coords.BoundingBox], resolution: float
     return rio.coords.BoundingBox(**max_bounds)
 
 
-def merge_rasters(rasters: list[gu.georaster.Raster], reference: Union[int, gu.Raster] = 0,
-                  merge_algorithm: Union[Callable, list[Callable]] = np.nanmean,
+def stack_rasters(rasters: list[gu.georaster.Raster], reference: Union[int, gu.Raster] = 0,
                   resampling_method: Union[str, rio.warp.Resampling] = "bilinear",
-                  use_ref_bounds = False) -> gu.georaster.Raster:
+                  use_ref_bounds = False) -> tuple[gu.georaster.Raster, rio.coords.BoundingBox]:
```

Comment:
I know that you've merged now, but why is it returning the bounds? What about `Raster.bounds` ??

Reply:
My mistake, it's actually returning a np.ndarray, as stated in the docstring, but wrongly indicated in the type.
Now wondering which one makes most sense, returning an array with references, or a Raster... What do you think?

Reply:
Right! Maybe it's worth returning a Raster? Since we have the shiny Raster.split_bands() functionality, I think it makes sense!

Reply:
Implemented in 001f86a